src/dependency-core/Core.Tests/Tests.Dependency.01/Obsolete/Obsolete.Create.cs

```diff
@@ -0,0 +1,54 @@
+using PrimeFuncPack.UnitTest;
+using Xunit;
+
+namespace PrimeFuncPack.Tests
+{
+    partial class OneDependencyTest
+    {
+        [Fact]
+        public void Obsolete_Create_ExpectMethodIsObsolete()
+        {
+            var type = typeof(Dependency<StructType>);
+            var methodName = nameof(Dependency<StructType>.Create);
+
+            var methodTypes = new[]
+            {
+                typeof(Func<IServiceProvider, StructType>)
+            };
+
+            var method = type.GetPublicStaticMethodOrThrow(methodName, 0, methodTypes);
+            var obsoleteAttribute = method.GetObsoleteAttributeOrThrow();
+
+            Assert.False(obsoleteAttribute.IsError);
+
+            var expectedInsteadMethodName = "Dependency<T>.From";
```

Comment:
Hereinafter: Let's rename to `expectedNewMethodName`

Reply:
Done